format/view-spec.md

```diff
@@ -0,0 +1,264 @@
+| Required | version-log | A list of timestamp and version ID pairs that encodes changes to the current version for the view. Each time the current-version-id is changed, a new entry should be added with the last-updated-ms and the new current-version-id. |
+| Optional | schemas | A list of schemas, the same as the ‘schemas’ field from Iceberg table spec. |
+| Optional | current-schema-id | ID of the current schema of the view |
+
+#### Versions
+
+Field "versions" is an array of structs with the following fields:
+
+| Required/Optional | Field Name | Description |
+|-------------------|------------|-------------|
+| Required | version-id | Monotonically increasing id indicating the version of the view. Starts with 1. |
+| Required | timestamp-ms | Timestamp expressed in ms since epoch at which the version of the view was created. |
+| Required | summary | A string map summarizes the version changes, including `operation`, described in [Summary](#summary). |
+| Required | representations | A list of "representations" as described in [Representations](#representations). |
+
+#### Version Log
+
+Field “version-log” is an array of structs that describe when each version was considered "current". Creation time is different and is stored in each version's metadata. This allows you to reconstruct what someone would have seen at some point in time. If the view has been updated and rolled back, this will show it. The struct has the following fields:
+
+| Required/Optional | Field Name | Description |
+|-------------------|------------|-------------|
+| Required | timestamp-ms | Tthe timestamp hen the referenced version was made the current version |
+| Required | version-id | Version id of the view  |
+
+#### Summary
+
+Field “summary” is a string map with the following keys. Only `operation` is required. Engines may store additional key-value pairs in this map.
+
+| Required/Optional | Key | Value |
+|-------------------|-----|-------|
+| Required | operation | A string value indicating the view operation that caused this metadata to be created. Allowed values are “create” and “replace”. |
+| Optional | engine-version | A string value indicating the version of the engine that performed the operation |
+
+#### Representations
+
+Each representation is stored as an object with only one common field "type".
+The rest of the fields are interpreted based on the type.
+There is only one type of representation defined in the spec.
+
+##### Original View Definition in SQL
+
+This type of representation stores the original view definition in SQL and its SQL dialect.
+
+| Required/Optional | Field Name | Description |
+|-------------------|------------|-------------|
+| Required | type | A string indicating the type of representation. It is set to "sql" for this type. |
+| Required | sql | A string representing the original view definition in SQL |
```

Comment:
Are we requiring that this is a `SELECT` statement? I don't think we want this set to `CREATE VIEW ... AS SELECT` (the full create/replace statement).